index.html

```diff
@@ -0,0 +1,462 @@
+              <li class="tech-used">Ruby</li>
+            </ul>
+            <button class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <div class="background">
+          <div class="cards">
+            <div class="card-title">
+              <p>Professional Art</p>
+              <p>Printing Data</p>
+            </div>
+            <p class="card-note">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="lang-tech">
+              <li class="tech-used">HTML</li>
+              <li class="tech-used">Bootstrap</li>
+              <li class="tech-used">Ruby</li>
+            </ul>
+            <button class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <div class="background">
+          <div class="cards">
+            <div class="card-title">
+              <p>Professional Art</p>
+              <p>Printing Data</p>
+            </div>
+            <p class="card-note">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="lang-tech">
+              <li class="tech-used">HTML</li>
+              <li class="tech-used">Bootstrap</li>
+              <li class="tech-used">Ruby</li>
+            </ul>
+            <button class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <div class="background">
+          <div class="cards">
+            <div class="card-title">
+              <p>Professional Art</p>
+              <p>Printing Data</p>
+            </div>
+            <p class="card-note">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="lang-tech">
+              <li class="tech-used">HTML</li>
+              <li class="tech-used">Bootstrap</li>
+              <li class="tech-used">Ruby</li>
+            </ul>
+            <button class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <div class="background">
+          <div class="cards">
+            <div class="card-title">
+              <p>Professional Art</p>
+              <p>Printing Data</p>
+            </div>
+            <p class="card-note">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="lang-tech">
+              <li class="tech-used">HTML</li>
+              <li class="tech-used">Bootstrap</li>
+              <li class="tech-used">Ruby</li>
+            </ul>
+            <button class="see-project-btn">See Project</button>
+          </div>
+        </div>
```

Comment:
- [ ] As required for the project to sick as much as possible to the your chosen Figma template design. kindy ensure that for each card the `See Project` button appear when the card is hovered and with a border in each card and also the button should be styled correctly and follow the interaction styling such as `active`, `hover`,....
### Figma design 
![Screenshot from 2023-05-12 13-06-52](https://github.com/meryemsanem/desktop-version/assets/83042742/81c9cc63-1534-40df-a3cd-b11870e5d02a)

## Your Design
![Screenshot from 2023-05-12 13-13-31](https://github.com/meryemsanem/desktop-version/assets/83042742/8214cc18-8ed6-44d7-b1b8-eb5e58e19a4c)